### ISSUE TITLE
Auto Complete: Selection after tpying and pressing Enter 

### ISSUE THREAD

**Issue by TomTomRixRix** [FachschaftMathPhys/moozean#80]
In all Auto Complete field, mainly the **Studierendes** and **Ordner** fields, it would be nice if one could simply type in an item and select it by pressing Enter in the end. This should obviously only work when the corresponding item exists in the list. Especially when selecting folder this would be really convenient as one would only tpye ''KM1A" + "Enter" and does not need to look at the screen at all and find the right item.

**Comment by SinForest** [comment]
Also, like on many other websites, the folder name should be confimable via the `,` key.

**Comment by samuscherer** [comment]
Alternatively, we could use semicolons instead of commas.

**Comment by henrixapp** [comment]
Help wanted. Not supported by ember-paper yet. https://github.com/miguelcobain/ember-paper/issues/871

**Comment by Sedrunum** [comment]
We are waiting for external help.

**Comment by Sedrunum** [comment]
@xomaczar Does this also work if I use the `search` function to create my options' list for the autocomplete field which queries the remote database for matching keywords? 

My approach so far:

save the queried results to a variable (studentoptions) inside the controller 
-> add `defaultHighlighted=(take 1 studentoptions)` to the autocomplete tag

Sadly I haven't observed any difference - the first entry is still not highlighted.

**Comment by Sedrunum** [comment]
(take 1 studentoptions) is part of the `ember-composable-helper` package

**Comment by Sedrunum** [comment]
From my understanding the `defaultHighlighted` option only highlights the specified entry when the list is initially opened. If I modify the provided options to select from the defaultHighlighted option wont help me focusing/highlighting the first entry since I have already abandoned the initial state.

**Comment by xomaczar** [comment]
Yeah it should work -`defaultHighlighted` should be a function that returns an item that you would like to highlight. `take` helper returns an item instead of a func.

**Comment by xomaczar** [comment]
What version of ember-paper are you using?

**Comment by Sedrunum** [comment]
1.0.0-beta.7

**Comment by Sedrunum** [comment]
tried it, same result
